contrib/grpc_metrics/src/main/java/io/opencensus/contrib/grpc/metrics/RpcMeasureConstants.java

```diff
@@ -25,7 +25,9 @@
 public final class RpcMeasureConstants {
 
   // Rpc tag keys.
+  /** Tag key that represents a RPC canonical status. */
```

Comment:
I think this should say gRPC, since it isn't a general RPC status.  It might help to add a link to the Status class.

Reply:
grpc_metrics doesn't depend on gRPC, so I cannot directly link to Status.java. I added a link to gRPC status doc instead.